crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -19,14 +19,17 @@ pub enum TupleParentheses {
     Default,
     /// Effectively `Some(Parentheses)` in `Option<Parentheses>`
     Expr(Parentheses),
```

Comment:
I don't think this variant is used anymore, it would make sense to remove it and replace the one usage with `Default` or not passing the option (which also sets `Default`)

Reply:
d7f0c26a3ac870caba44b80a0622adbb217ecbc2

I chose 
> not passing the option